css/default.css

```diff
@@ -1319,6 +1323,11 @@ button.loginButton {
     color: var(--widget-text);
 }
 
+/* Hiding the highlighter-rouge dark mode toggle */
```

Comment:
Please leave a comment to share context (perhaps with the Jira ticket number), or a to-do comment to eventually re-enable it.